From: Thomas Moulard <thomas.moulard@gmail.com>
Date: Sat, 30 Nov 2013 16:16:13 +0900
Subject: visp-config should rely on pkg-config information

Make visp-config arch-independent by relying on pkg-config information.

Author: Thomas Moulard <thomas.moulard@gmail.com>
Forwarded: yes
---
 CMakeModules/visp-config.install.in | 18 ++++++------------
 1 file changed, 6 insertions(+), 12 deletions(-)

diff --git a/CMakeModules/visp-config.install.in b/CMakeModules/visp-config.install.in
index e797f26..88504df 100755
--- a/CMakeModules/visp-config.install.in
+++ b/CMakeModules/visp-config.install.in
@@ -48,20 +48,14 @@ relpath=`(cd $relpath/..; pwd)`

 PREFIX=$relpath

-CFLAGS="${VISP_CONFIG_CFLAGS_SCRIPT}"
+CFLAGS_CMD=`pkg-config --cflags visp`
+CFLAGS=$CFLAGS_CMD

-LIBS="${VISP_CONFIG_LIBS_SCRIPT}"
+LIBS_CMD=`pkg-config --libs visp`
+LIBS=$LIBS_CMD

-VERSION="${VISP_VERSION}"
-
-#CFLAGS_CMD=`pkg-config --cflags visp`
-#CFLAGS=$CFLAGS_CMD
-
-#LIBS_CMD=`pkg-config --libs visp`
-#LIBS=$LIBS_CMD
-
-#VERSION_CMD=`pkg-config --modversion visp`
-#VERSION=$VERSION_CMD
+VERSION_CMD=`pkg-config --modversion visp`
+VERSION=$VERSION_CMD

 NO_NEWLINE_CHARACTER="${VISP_ECHO_NO_NEWLINE_CHARACTER}"
 NO_NEWLINE_OPTION="${VISP_ECHO_NO_NEWLINE_OPTION}"
